srcpkgs/pantheon-default-settings/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'pantheon-default-settings'
+pkgname=pantheon-default-settings
+version=5.1.2
+revision=1
+build_style=fetch
+hostmakedepends="tar"
```

Comment:
you shouldn't need these two lines. Without `build_style` it expects `do_install()`. Exactly what you need.

Reply:
Done!